tools/README.md

```diff
@@ -0,0 +1,66 @@
+# Performance testing tool for ballerina-message-broker 
+
+This tool allows ballerina message broker developers to get an idea about the performance of the message broker after a modification is done in the code.Following criteria is going to be used in evaluating the performance.
+
+- Connection Load  - The number of message producers, or message consumers, or the number of concurrent connections a system can support.
+- Message throughput - The number of messages or message bytes that can be pumped through a messaging system per second.
+- Latency - The time it takes a particular message to be delivered from message producer to message consumer.
+
+## How to use 
+
+1.Copy following jars to JMETER_HOME/lib.
+
+- andes-client-0.13.wso2v8.jar
+- geronimo-jms_1.1_spec-1.1.0.wso2v1.jar
+- slf4j-1.5.10.wso2v1.jar
+
+2.Create jndi.properties file
+
+```properties
+# register some connection factories
+# connectionfactory.[jndiname] = [ConnectionURL]
+connectionfactory.QueueConnectionFactory = amqp://admin:admin@clientID/carbon?brokerlist='tcp://localhost:<port>'
+ 
+# register some queues in JNDI using the form
+# queue.[jndiName] = [physicalName]
+queue.QueueName = <queue_name>
+```
+
+3.Download the jmx file in test_plan folder to your local machine
+
+4.Create a properties files including following details.
+```properties
+jmeter_home = <jmeter_home>
+jmx_file_location =<jmx_file_location>
+jndi_file_location= <jndi_file_location>
```

Comment:
can we set defaults to 'jmx_file_loation' and 'jndi_file_location' too? Could be the jmeter home?

Reply:
- can we set defaults to 'jmx_file_loation' and 'jndi_file_location' too?
 Created a folder named resources inside tools/scripts and included default jmx,jndi files.

Added logic to use jmeter home from environment variables if it's not specified in the properties file.